setup.py

```diff
@@ -4,11 +4,14 @@
 from setuptools import setup, find_packages
 from os         import path
 
+import cvmfs
+
+
 readme_path = path.join(path.dirname(__file__), 'README')
 
 setup(
-  name="python-cvmfsutils",
-  version="0.2.0",
+  name=cvmfs.package_name,
+  version=cvmfs.version,
```

Comment:
I have just installed cvmfsutils (and not python-cvmfsutils) from the scratch and it worked like a charm. Here's the command:
```
pb-d-128-141-170-35:python-cvmfsutils guimard$ sudo !!
sudo python setup.py install
Password:
running install
Checking .pth file support in /Library/Python/2.7/site-packages/
/usr/bin/python -E -c pass
TEST PASSED: /Library/Python/2.7/site-packages/ appears to support .pth files
running bdist_egg
running egg_info
creating cvmfsutils.egg-info
writing requirements to cvmfsutils.egg-info/requires.txt
writing cvmfsutils.egg-info/PKG-INFO
writing top-level names to cvmfsutils.egg-info/top_level.txt
writing dependency_links to cvmfsutils.egg-info/dependency_links.txt
writing manifest file 'cvmfsutils.egg-info/SOURCES.txt'
reading manifest file 'cvmfsutils.egg-info/SOURCES.txt'
reading manifest template 'MANIFEST.in'
writing manifest file 'cvmfsutils.egg-info/SOURCES.txt'
installing library code to build/bdist.macosx-10.11-intel/egg
running install_lib
running build_py
copying cvmfs/__init__.py -> build/lib/cvmfs
copying cvmfs/cache.py -> build/lib/cvmfs
copying cvmfs/history.py -> build/lib/cvmfs
copying cvmfs/repository.py -> build/lib/cvmfs
copying cvmfs/revision.py -> build/lib/cvmfs
copying cvmfs/test/repository_test.py -> build/lib/cvmfs/test
creating build/bdist.macosx-10.11-intel/egg
creating build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/__init__.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/_common.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/_exceptions.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/availability.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/cache.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/catalog.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/certificate.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/dirent.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/fetcher.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/history.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/manifest.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/repository.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/revision.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
copying build/lib/cvmfs/root_file.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
creating build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/__init__.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/__main__.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/certificate_test.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/file_sandbox.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/manifest_test.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/md5_handling_test.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/mock_repository.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/repository_test.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/test/whitelist_test.py -> build/bdist.macosx-10.11-intel/egg/cvmfs/test
copying build/lib/cvmfs/whitelist.py -> build/bdist.macosx-10.11-intel/egg/cvmfs
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/_common.py to _common.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/_exceptions.py to _exceptions.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/availability.py to availability.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/cache.py to cache.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/catalog.py to catalog.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/certificate.py to certificate.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/dirent.py to dirent.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/fetcher.py to fetcher.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/history.py to history.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/manifest.py to manifest.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/repository.py to repository.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/revision.py to revision.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/root_file.py to root_file.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/__init__.py to __init__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/__main__.py to __main__.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/certificate_test.py to certificate_test.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/file_sandbox.py to file_sandbox.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/manifest_test.py to manifest_test.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/md5_handling_test.py to md5_handling_test.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/mock_repository.py to mock_repository.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/repository_test.py to repository_test.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/test/whitelist_test.py to whitelist_test.pyc
byte-compiling build/bdist.macosx-10.11-intel/egg/cvmfs/whitelist.py to whitelist.pyc
creating build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying cvmfsutils.egg-info/PKG-INFO -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying cvmfsutils.egg-info/SOURCES.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying cvmfsutils.egg-info/dependency_links.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying cvmfsutils.egg-info/requires.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
copying cvmfsutils.egg-info/top_level.txt -> build/bdist.macosx-10.11-intel/egg/EGG-INFO
zip_safe flag not set; analyzing archive contents...
creating 'dist/cvmfsutils-0.3.0-py2.7.egg' and adding 'build/bdist.macosx-10.11-intel/egg' to it
removing 'build/bdist.macosx-10.11-intel/egg' (and everything under it)
Processing cvmfsutils-0.3.0-py2.7.egg
Copying cvmfsutils-0.3.0-py2.7.egg to /Library/Python/2.7/site-packages
Adding cvmfsutils 0.3.0 to easy-install.pth file

Installed /Library/Python/2.7/site-packages/cvmfsutils-0.3.0-py2.7.egg
Processing dependencies for cvmfsutils==0.3.0
Searching for M2Crypto==0.22.3
Best match: M2Crypto 0.22.3
M2Crypto 0.22.3 is already the active version in easy-install.pth

Using /usr/local/lib/python2.7/site-packages
Searching for requests==2.6.0
Best match: requests 2.6.0
requests 2.6.0 is already the active version in easy-install.pth

Using /usr/local/lib/python2.7/site-packages
Searching for python-dateutil==2.4.2
Best match: python-dateutil 2.4.2
python-dateutil 2.4.2 is already the active version in easy-install.pth

Using /usr/local/lib/python2.7/site-packages
Searching for six==1.9.0
Best match: six 1.9.0
six 1.9.0 is already the active version in easy-install.pth

Using /usr/local/lib/python2.7/site-packages
Finished processing dependencies for cvmfsutils==0.3.0
```